### PR TITLE
Add simulcast support table

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**
#1784 

**Description of changes:**
- Documentation guide update.

**Testing:**
- Could not test webview but tested Samsung Internet in SauceLabs real device. video shows black but still could see no errors in devtools.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
NA

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
NA.

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
NA.

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: guides/05_Simulcast.md [hunk 1]
```diff
@@ -18,7 +18,15 @@ Simulcast support is built into the WebRTC library in the majority of browsers.
...
+The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors. Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+
+### Browser compatibility
+
+|Browser              |Supported versions  |
+|---                  |---                 |
+|Chrome               |76+                 |
+
+> Note: Simulcast in Amazon Chime SDK for JavaScript should also work in Chromium-based browsers on desktop like Chromium Edge and Opera; and on mobile like Chromium-Webview and Samsung Internet.
```
Comment (hensmi-amazon) [inline guides/05_Simulcast.md:29]: Is it possible to get the actual version numbers of those browsers at the M76 point (maybe we can estimate too).

File: guides/05_Simulcast.md [hunk 2]
```diff
@@ -18,7 +18,15 @@ Simulcast support is built into the WebRTC library in the majority of browsers.
...
 
-The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors.   Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors. Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+
+### Browser compatibility
+
+|Browser              |Supported versions  |
+|---                  |---                 |
+|Chrome               |76+                 |
```
Comment (ltrung) [inline guides/05_Simulcast.md:27]: If we want to add a support table, we want a full coverage of what Chromium browser we support (Chrome, Android Chrome, Edge, etc). :)

File: guides/05_Simulcast.md (hunk 1 above)
Comment (richnew10) [reply guides/05_Simulcast.md:29]: Our documentation should not routinely say "should work". Either we've tested it and it works (or doesn't), or it should be an uncommon case that most customers won't care about. I would argue that Edge is in the first set, not the second.

Comment (devalevenkatesh) [reply guides/05_Simulcast.md:29]: Agree, I checked with internal QA around this and tested few but "should work" is not ideal, and will need more time to actually get all those browsers tested as mentioned here with the list supported version thanks! I have created an internal ticket to track this work and add support for Simulcast like we have in VF guide.

**Comment by devalevenkatesh** [comment]
Closing PR and will re-open later as it needs more testing on specific browsers with versions supported by Amazon Chime SDK for JavaScript. Tracking in an internal ticket.

**Pull request closed without merging**